aie_runtime_lib/AIE2/lut_based_ops.h

```diff
@@ -81,8 +81,8 @@ __attribute__((always_inline)) bfloat16 getInvBf16(float x) {
   return *inv_x;
 }
 
-extern float tanh_lut_ab[];
-extern float tanh_lut_cd[];
+alignas(aie::vector_decl_align) extern float tanh_lut_ab[128];
+alignas(aie::vector_decl_align) extern float tanh_lut_cd[128];
```

Comment:
Is there any benefit to having the alignment and size of these arrays declared here?